simple/schema.py

```diff
@@ -268,3 +270,41 @@ class ModeledParameters(Base):
     unit = Column(String(20))
     comments = Column(String(1000))
     reference = Column(String(30), ForeignKey('Publications.reference', onupdate='cascade'), primary_key=True)
+
+
+# -------------------------------------------------------------------------------------------------------------------
+# Views
+
+ParallaxView = view(
+    "ParallaxView",
+    Base.metadata,
+    sa.select(
+        Parallaxes.source.label('source'),
+        Parallaxes.parallax.label('parallax'),
+        Parallaxes.parallax_error.label('parallax_error'),
+        (1000./Parallaxes.parallax).label('distance'),  # distance in parsecs
+        Parallaxes.comments.label('comments'),
+        Parallaxes.reference.label('reference')
+    ).select_from(Parallaxes)
+    .where(sa.and_(Parallaxes.adopted == True, Parallaxes.parallax > 0)),
+)
+
+PhotometryView = view(
+    "PhotometryView",
+    Base.metadata,
+    sa.select(
+        Photometry.source.label('source'),
+        sa.func.avg(sa.case((Photometry.band == "2MASS.J", Photometry.magnitude))).label("2MASS.J"),
```

Comment:
No optical? :(

Reply:
At some point we can add more, but this is a proof-of-concept. Feel free to add in the future ;)